src/components/Input/DropDown/DropDownGroup.js

```diff
@@ -0,0 +1,271 @@
+    }
+  }
+`;
+
+const StyledChildWrapper = styled.div`
+  display: none;
+  background-color: white;
+  border-radius: 0 0 2px 2px;
+  border: solid 1px ${colors.onyx.muted};
+  padding-top: 4px;
+  padding-bottom: 8px;
+  white-space: nowrap;
+  margin-top: -1px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.12);
+  min-width: 100%;
+  box-sizing: content-box;
+
+  &.dropDown--clicked {
+    flex-direction: column;
+    position: absolute;
+    overflow-y: auto;
+    flex: 1;
+    display: flex;
+    overflow: auto;
+    height: 135px;
+  }
+`;
+
+const StyledGroupWrapper = styled.div`
+  position: relative;
+  color: grey;
+  width: 100%;
+`;
+
+const StyledChevron = styled(ChevronIcon)`
+  margin-right: 12px;
+
+  &.dropDown__icon--hide {
+    display: none;
+  }
+`;
+
+const StyledSelectedText = styled.div`
+  font-size: 16px;
+  white-space: nowrap;
+
+  &.dropDown--border {
+    margin-left: 15px;
+    width: 60%;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+
+  &.dropDown--no-border {
+    margin-right: 8px;
+    width: 60%;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+  &.dropDown--alignment {
+    margin-right: 32px;
+  }
+`;
+
+const StyledKeyboardProvider = styled(KeyBoardProvider)`
+  min-height: min-content;
+  flex-direction: column;
+  display: flex;
+`;
+
+class DropDownGroup extends React.Component {
+  componentDidMount() {
+    document.addEventListener("click", this.handleOutsideClick);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener("click", this.handleOutsideClick);
+  }
+
+  onClick = () => {
+    this.setState(({ isOpen }) => ({ isOpen: !isOpen }));
+  };
+
+  onKeyDown = e => {
+    if (e.keyCode === ARROWUP || e.keyCode === ARROWDOWN) {
+      e.preventDefault();
+      this.setState({ isOpen: true });
+    }
+    if (e.keyCode === SPACEBAR) {
+      this.setState(({ isOpen }) => ({ isOpen: !isOpen }));
+    }
+  };
+
+  getCurrentSelection = value => {
+    const [label = null] = React.Children.toArray(this.props.children).filter(
+      c => c.props.value === value
+    );
+
+    if (label) {
+      return label.props.children;
+    }
+    return label;
+  };
+
+  handleOutsideClick = e => {
+    if (!this.thisComponent.current.contains(e.target)) {
+      this.setState(() => ({ isOpen: false }));
+    }
+  };
+
+  displayLabel = (selected, variant) => {
+    if (variant === 0 && selected.length === 0) return selected.length === 0;
+
+    if (variant === 0 && selected.length > 0)
+      return this.getCurrentSelection(selected[0]);
+
+    return `${this.props.label} ${this.getCurrentSelection(selected[0])}`;
+  };
+
+  /* eslint-disable */
+  state = {
+    isOpen: false,
+    onClose: this.onClick
+  };
+  /* eslint-enable */
+
+  thisComponent = React.createRef();
+
+  render() {
+    const { children, value, onChange, variant, ...props } = this.props;
+    return (
+      <SelectionProvider
+        onChange={onChange}
+        isMultiSelect={false}
```

Comment:
Should multiselect be configurable?

Reply:
It is configurable for the Selection Provider but drop down group should only be able to select on value at a time.